src/main/java/uk/gov/companieshouse/presenter/account/validation/utils/StringValidator.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.companieshouse.presenter.account.validation.utils;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+
+public final class StringValidator {
+
+    private static final int MAX_REGEX_LENGTH = 200;
+
+    private static final String VALID_CHARACTERS = "-,.:; 0-9A-Z&@$£¥€'\"«»?!/\\\\()\\[\\]{}<>*=#%+ÀÁÂÃÄÅĀĂĄÆǼÇĆĈĊČÞĎÐÈÉÊËĒĔĖĘĚĜĞĠĢĤĦÌÍÎÏĨĪĬĮİĴĶĹĻĽĿŁÑŃŅŇŊÒÓÔÕÖØŌŎŐǾŒŔŖŘŚŜŞŠŢŤŦÙÚÛÜŨŪŬŮŰŲŴẀẂẄỲÝŶŸŹŻŽa-zſƒǺàáâãäåāăąæǽçćĉċčþďðèéêëēĕėęěĝģğġĥħìíîïĩīĭįĵķĺļľŀłñńņňŋòóôõöøōŏőǿœŕŗřśŝşšţťŧùúûüũūŭůűųŵẁẃẅỳýŷÿźżž";
+
+    /**
+     * 
+     * @param unvalidatedString the raw string to be validated
+     * @param maxLength of a valid string
+     * @return Optional of string that passed validation or an empty optional if not.
+     * @throws IllegalArgumentException if an invalid max length is entered.
+     */
+    public static final Optional<String> validateString(final String unvalidatedString, final int maxLength) throws IllegalArgumentException {
```

Comment:
Instead of returning an optional, as this function is doing a check, could it not return true or false